keyboards/exent/config.h

```diff
@@ -19,8 +19,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "config_common.h"
 
-#define VENDOR_ID 0x20A0
-#define PRODUCT_ID 0x422D
+#define VENDOR_ID 0x5143  // QC
+#define PRODUCT_ID 0x4558 // EX 
 #define DEVICE_VER 0x0001
 #define MANUFACTURER Quadcube
 #define PRODUCT Exent
```

Comment:
While we're here:
```suggestion
#define VENDOR_ID    0x5143 // "QC"
#define PRODUCT_ID   0x4558 // "EX"
#define DEVICE_VER   0x0001
#define MANUFACTURER Quadcube
#define PRODUCT      Exent
```
and `DESCRIPTION` can be removed.